packages/ketcher-react/src/infrastructure/services/struct/remoteStructService.ts

```diff
@@ -87,22 +87,22 @@ class IndigoService implements StructService {
   }
 
   async info(): Promise<any> {
-    let indigoVersion: any,
-      imagoVersions: any,
-      isAvailable: boolean = false
-    try {
-      const response = await request('GET', this.baseUrl + 'info')
-      indigoVersion = response['indigo_version']
-      imagoVersions = response['imago_versions']
-      isAvailable = true
-    } catch (e) {
-      isAvailable = false
-    }
+    // let indigoVersion: any,
```

Comment:
if you  want to  mock structService you need to implement StructService interface, cretae your own StructServiceProvider and update your local version of App.tsx. please look at StandaloneStructService for example: 
```js
import { RemoteStructServiceProvider } from 'ketcher-core'
import 'ketcher-react/dist/index.css'
;(global as any).Miew = Miew

let structServiceProvider: any = new RemoteStructServiceProvider(
  process.env.REACT_APP_API_PATH!
)
if (process.env.MODE === 'standalone') {
  const { StandaloneStructServiceProvider } = require('ketcher-standalone')
  structServiceProvider = new StandaloneStructServiceProvider()
}

const App = () => {
  return (
    <Editor
      staticResourcesUrl={process.env.PUBLIC_URL}
      structServiceProvider={structServiceProvider}
    />
  )
}
```